scripts/influxdb2-upgrade.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Environment defaults
+INFLUXD_CONFIG_PATH=/etc/influxdb/config.toml
+INFLUXD_BOLT_PATH=/var/lib/influxdb/influxd.bolt
+INFLUXD_ENGINE_PATH=/var/lib/influxdb/engine
+
+export INFLUXD_CONFIG_PATH INFLUXD_BOLT_PATH INFLUXD_ENGINE_PATH
+
+# Check upgrade status
+bolt_dir="/root/.influxdbv2 /var/lib/influxdb/.influxdbv2 /var/lib/influxdb"
+for bolt in $bolt_dir
+do
+  if [[ -s ${bolt}/influxd.bolt ]]; then
+    echo "An existing ${bolt}/influxd.bolt file was found indicating InfluxDB is"
+    echo "already upgraded to v2.  Exiting."
+    exit 1
+  fi
+done
+
+# Perform upgrade
+sudo /usr/bin/influxd upgrade \
+  --config-file=/etc/influxdb/influxdb.conf \
+  --v2-config-path=${INFLUXD_CONFIG_PATH} \
+  -m $INFLUXD_BOLT_PATH -e $INFLUXD_ENGINE_PATH
+
+if [[ $? -eq 0 ]]; then
+cat << EOF
+
+The upgrade completed successfully.  Execute the following to start InfluxDB:
+
+sudo systemctl start influxdb
+EOF
+fi
```

Comment:
Should we have a message + early exit here for the failure case? Upgrade isn't idempotent, so it could be nice to print an `rm` command for cleanup in the case of failure.

Reply:
Most recent commit expands the upgrade script for both pass/fail cases, and outputs a respective v1 or v2 data removal script.  They need to be double-checked to make sure it's removing only what's necessary.